### PR TITLE
Added options to match Trello card - Dashboard Update, removed extra …

### PR DESCRIPTION
**Pull request by Rrawla2**
…comma from index.js

# Description

Added DashboardUpdate Trello card options for custom workout plans, Profile, custom meal plans - buttons only with links to "/".  Removed an extra comma from index.js that was showing up on the rendered page.  No styling done for this component.

## Change Status

- [X] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by onepeso — APPROVED** [review]
_Great catching that comma!_

File: src/components/dashboard/Dashboard.js [hunk 1]
```diff
@@ -10,11 +10,30 @@ const Dashboard = () => {
...
+      <button type='submit' className="dashboard_btn" onClick={() => history.push("/")}>
+        Find a prefab plan
+      </button>
+      </div>
+      <div>
+      <button type='submit' className="dashboard_btn" onClick={() => history.push("/")}>
+        Create custom meal plan
+      </button>
+      <button type='submit' className="dashboard_btn" onClick={() => history.push("/")}>
+        Find a prefab meal plan
```
Comment (jess-lam) [inline src/components/dashboard/Dashboard.js:26]: @Rrawla2 Is there supposed to be two buttons for Find a prefab meal plan?

Comment (Rrawla2) [reply src/components/dashboard/Dashboard.js:26]: Only 1.  I can rename the other to prefab workout plan instead of prefab plan

Comment (jess-lam) [reply src/components/dashboard/Dashboard.js:26]: Oh I see. I got confused. Yeah. I think renaming the other to prefab workout plan would be more distinguishable. Thank you!

Comment (Rrawla2) [reply src/components/dashboard/Dashboard.js:26]: Ok fixed the button text and pushed.  Thanks for catching that mistake!

Comment (jess-lam) [reply src/components/dashboard/Dashboard.js:26]: Thank you! :)

**Review by jess-lam — APPROVED** [review]
Great edit!

**Pull request merged**